### PR TITLE
Fix sidebar colors for kitty terminal documention

### PR DESCRIPTION
**Pull request by icewind1991**
Fixes the background colors of the kitty terminal documentation (https://sw.kovidgoyal.net/kitty/)

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -13713,6 +13713,19 @@ a[class^="PaginationButton"]:hover {
...
+
+sw.kovidgoyal.net
+
+CSS
+.sidebar-drawer, .sidebar-search {
+  background: var(--darkreader-neutral-background) !important;
+}
+.sidebar-tree a.current, .sidebar-tree label:hover {
+  background: var(--darkreader-selection-background) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13727]: What is the purpose of these 2 selectors?

Comment (icewind1991) [reply src/config/dynamic-theme-fixes.config:13727]: it's for the background of the selected sidebar item and the ⌃/⌄arrows for expand/collapse when hovering over them

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -13713,6 +13713,19 @@ a[class^="PaginationButton"]:hover {
     background-color: var(--darkreader-bg--nyh-color-white);
 }
 
+
+================================
+
+sw.kovidgoyal.net
+
+CSS
+.sidebar-drawer, .sidebar-search {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13722]:
```suggestion
.sidebar-drawer,
.sidebar-search {
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -13713,6 +13713,19 @@ a[class^="PaginationButton"]:hover {
...
+  background: var(--darkreader-neutral-background) !important;
+}
+.sidebar-tree a.current, .sidebar-tree label:hover {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13725]:
```suggestion
.sidebar-tree a.current,
.sidebar-tree label:hover {
```

**Review by Gusted — APPROVED** [review]

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -13713,6 +13713,21 @@ a[class^="PaginationButton"]:hover {
     background-color: var(--darkreader-bg--nyh-color-white);
 }
 
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:13716]:
```suggestion
```

**Pull request merged**